### DISCUSSION TITLE
Prometheus connection refused

### DISCUSSION THREAD

**Discussion by Gabs22LB** [netbox-community/netbox-docker#748] [Getting Help]
Hi, I am configuring my netbox on docker, I am configuring prometheus but it can't connect to my netbox, do you have a lead I could explore? 

I tried different endpoint but nothings works for now

![image](https://user-images.githubusercontent.com/103570230/163153840-a53bd599-d3a4-4445-adfa-29772871229a.png)

**Comment by rfdrake** [comment]
I know it's three years later, but I'm posting here in case someone else is looking for help. I've updated the wiki with a new config that works for me:

https://github.com/netbox-community/netbox-docker/wiki/Monitoring
